build/lib/locFunc.js

```diff
@@ -18,6 +18,13 @@ const File = require("vinyl");
 const rimraf = require("rimraf");
 const gulp = require("gulp");
 const vfs = require("vinyl-fs");
+/**
```

Comment:
these files are from VSCode, right? if so, we should add the sql carbon edit comment, right?

Reply:
the locFunc file is an original addition that I created a while back, so it does not need a carbon edit. @alanrenmsft